packages/create-svelte/templates/default/src/hooks.ts

```diff
@@ -1,11 +1,10 @@
-import { v4 as uuid } from '@lukeed/uuid';
 import type { Handle } from '@sveltejs/kit';
 import * as cookie from 'cookie';
 
 /** @type {import('@sveltejs/kit').Handle} */
 export const handle: Handle = async ({ event, resolve }) => {
 	const cookies = cookie.parse(event.request.headers.get('cookie') || '');
-	event.locals.userid = cookies['userid'] || uuid();
+	event.locals.userid = cookies['userid'] || crypto.randomUUID();
```

Comment:
Are we doing something elsewhere that lets us rely on `crypto` as a global? I just tried that in another unrelated project, and it didn't work.

Reply:
Ah, I guess this is the new `polyfills.js`. Never mind!